test/spec/modules/sharedIdSystem_spec.js

```diff
@@ -1,77 +0,0 @@
-import {
```

Comment:
Why is this module test being removed?

Reply:
According to https://github.com/prebid/Prebid.js/issues/6640: 

> remove the following sharedid files
> 
> modules/sharedIdSystem.js
> test/spec/modules/sharedIdSystem_spec.js
> modules/sharedIdSystem.md

Reply:
updated